docs/pages/eas-update/continuous-deployment.mdx

```diff
@@ -63,6 +63,16 @@ To exclude files causing the differences, add them to the **.fingerprintignore**
 
 </Collapsible>
 
+### App version numbers
+
+When using continuous deployment version number incrementation must be done automatically. While the specifics will differ based on the preferred numbering scheme, the general strategy is:
+
+1. Before continuous deployment CI steps, check if a new release will be created (due to a new fingerprint).
+2. If so, bump the version number in the source code and push the version bump commit.
+3. Continue continuous deployment CI steps with the new commit.
+
+An example is provided in the [continuous-deploy-fingerprint GitHub Action documentation](https://github.com/expo/expo-github-action/tree/main/continuous-deploy-fingerprint#continuous-deployment-on-main-advanced).
```

Comment:
```suggestion
An example is provided in the [`continuous-deploy-fingerprint` GitHub Action documentation](https://github.com/expo/expo-github-action/tree/main/continuous-deploy-fingerprint#continuous-deployment-on-main-advanced).
```

Since this is an action name.